sentences/en/_common.yaml

```
@@ -372,6 +382,7 @@ expansion_rules:
   in: "(in|on|at|of)"
   position: "{position}[[ ]%| percent]"
   volume: "{volume:volume_level}[[ ]%| percent]"
+  media_players: "([media ]player[s]|speaker[s]stereo[s]|tv[s])"
```

Missing a `|` between speaker and stereo. Should maybe add `television`? Also, this could be simplified by moving the `[s]` to the end.

```suggestion
  media_players: "([media ]player|speaker|stereo|tv|television)[s]"
```

---

Agreed regarding `television`, but moving the `[s]` at the end makes it less readable and does not optimize matching.